mailbox/api/src/test/java/org/apache/james/mailbox/model/MailboxPathTest.java

```diff
@@ -125,6 +125,55 @@ public void sanitizeShouldPreserveRedundantDelimiters() {
                 MailboxPath.forUser("user", "a..a"));
     }
 
+    @Test
+    public void hasEmptyNameInHierarchyShouldBeFalseIfSingleLevelPath() {
+        assertThat(MailboxPath.forUser("user", "a")
+            .hasEmptyNameInHierarchy('.'))
+            .isFalse();
+    }
+
+    @Test
+    public void hasEmptyNameInHierarchyShouldBeFalseIfNestedLevelWithNonEmptyNames() {
+        assertThat(MailboxPath.forUser("user", "a.b.c")
+            .hasEmptyNameInHierarchy('.'))
+            .isFalse();
+    }
+
+    @Test
```

Comment:
you could have parametrized these tests

Reply:
I'm afraid to hide the intent of tests.
But it would be a good incentive to migrate this test to JUnit 5.